common/tools.py

```diff
@@ -1846,10 +1831,13 @@ def escapeIPv6Address(address):
     Returns:
         str:            ``address`` in square brackets
     """
-    if isIPv6Address(address):
-        return '[{}]'.format(address)
-    else:
-        return address
+    try:
+        ip = ipaddress.ip_address(address)
+        if ip.version == 6:
+            return '[{}]'.format(address)
+    except ValueError:
+        pass    # Address is not a valid IP address, so return it unchanged
```

Comment:
I wonder if we might lose some error cases when catching an exception here. I would prefer to see exceptions. Please remove the try-catch block.

```suggestion
    ip = ipaddress.ip_address(address)
    if ip.version == 6:
        return '[{}]'.format(address)
```